### ISSUE TITLE
System.InvalidOperationException on 64bit assembly

### ISSUE THREAD

**Issue by shadowua** [Washi1337/OldRod#18]
I was testing OldRod on some dummies and found out that it cannot reassemble 64bit net files. 

Here is log:

```
// ...
21:24:16.586 [Main]: Executing Clean up stage...
21:24:16.587 [Main]: Commiting changes to metadata streams...
21:24:16.790 [Main]: Not removing koi stream as some exports were ignored.
21:24:16.791 [Main]: Reassembling file...
21:24:16.812 [TUI]: Something went wrong! Try the latest version or report a bug
 at the repository.
21:24:16.819 [TUI]: System.InvalidOperationException: Sequence contains no matching element
// ...
```

It seems that is related to the fact that AsmResolver cannot find and properly write import directory to assembly as there is none in 64bit file. I have attached the file that I am working with.
[WindowsFormsApplication4.zip](https://github.com/Washi1337/OldRod/files/3725897/WindowsFormsApplication4.zip)

**Comment by Washi1337** [comment]
Yes currently this is indeed a limitation of AsmResolver 3. When I get the chance I will probably add this (soon^tm). Since this is not really related to OldRod, I will transfer this issue to the appropriate repository.

One thing to keep in mind however, is that AsmResolver 3 does not have priority as much anymore since development on v4 has started (https://github.com/Washi1337/AsmResolver/tree/v4).
